Src/Plugins/Tests.Security/SEC104.UseSecureApiTests.cs

```diff
@@ -19,5 +19,9 @@ public UseSecureApiTests(ITestOutputHelper outputHelper) : base(outputHelper)
         [Fact]
         public void UseSecureApi_EndToEndFunctionalTests()
             => RunAllTests();
+
+        [Fact]
+        public void UseSecureApi_ValidateOutputFiles()
+            => ValidateOutputFiles();
```

Comment:
>ValidateOutputFiles [](http://example.com/codeflow?start=15&length=19)

How are you going to guarantee the order of test execution? i.e., guarantee that all the output files have been generated?

Shouldn't we consider running validation, if it even occurs after that fact, in RunAllTests when we produce the log file?

Reply:
u are right.
just changed the logic.
thanks for reviewing.